### PR TITLE
Update emojis to latest version (15)

### PR DESCRIPTION
**Pull request by vjousse**
I'm using this library for an Elm Mastodon client https://github.com/n1k0/tooty and I wanted to have an updated emoji list.

So I made 2 python scripts for this purpose that parses the data from https://github.com/iamcal/emoji-data (for the emoji list) and https://github.com/unicode-org/cldr/releases (for the associated keywords / annotations) and generate the corresponding Elm code.

I took the opportunity to format the files with `elm-format` and to update the example using an `elm.json` file to make it work.

This is a first draft, don't hesitate if you have any questions/remarks.

And by the way, thanks for the amazing work. I may want to add an emoji search feature later on for Tooty. If so, I'll try to make a related PR here.

### CODE REVIEW

File: src/Types.elm [hunk 1]
```diff
@@ -1,21 +1,29 @@
-module Types exposing (Emoji, Category)
+module Types exposing (Category, Emoji)
 
 import Dict exposing (Dict)
 
+
+
 -- typedef for a category of emojis. in the
 -- main file, we render emojis by category.
```
Comment (turboMaCk) [inline src/Types.elm]: This comment should be turn into documentation comment.


```elm
{-| Comment -}
```

**Comment by turboMaCk** [comment]
@vjousse thanks for the PR and for reformatting code with elm-format. Before I look more closely into implementation can you please just skim through elm files and turn all comments describing functions into documentation comments so we get proper formatting? Just in case I need to open some PR to be merged into this one to avoid conflicts.

**Comment by vjousse** [comment]
> @vjousse thanks for the PR and for reformatting code with elm-format. Before I look more closely into implementation can you please just skim through elm files and turn all commencements describing functions into documentation comments so we get proper formatting? Just in case I need to open some PR to be merged into this one to avoid conflicts.

Sure, it should be good now.

**Review by turboMaCk — APPROVED** [review]
LGTM

**Pull request merged**